rpcs3/util/types.hpp

```diff
@@ -55,6 +55,48 @@ using namespace std::literals;
 #define AUDIT(...) (static_cast<void>(0))
 #endif
 
+namespace utils
+{
+	template <typename F>
+	struct fu_helper
```

Comment:
why not use fun helper and FUN

Reply:
is FU so bad?

Reply:
I would prefer fun helper over fuck you helper. Positivity is better and I don't think one additional letter is too big of a burden.

Reply:
Okay